Reqnroll/Infrastructure/TestExecutionEngine.cs

```diff
@@ -103,7 +103,16 @@ public virtual async Task OnTestRunStartAsync()
 
             if (_analyticsTransmitter.IsEnabled)
             {
-                _ = Task.Run(TryTransmitReqnrollProjectRunningEventAsync);
+                try
```

Comment:
There must be a merge issue here. It seems that the changes of #629 are not merged to the branch.

Reply:
Fixed.